packages/api/__tests__/specs/author.ts

```diff
@@ -0,0 +1,160 @@
+import {MongoDBAdapter} from '@wepublish/api-db-mongodb'
+import {ApolloServerTestClient} from 'apollo-server-testing'
+import {createGraphQLTestClientWithMongoDB} from '../utility'
+import {
+  CreateAuthor,
+  AuthorInput,
+  AuthorList,
+  Author,
+  UpdateAuthor,
+  DeleteAuthor
+} from '../api/private'
+
+let testClientPublic: ApolloServerTestClient
+let testClientPrivate: ApolloServerTestClient
+let dbAdapter: MongoDBAdapter
+
+beforeAll(async () => {
+  try {
+    const setupClient = await createGraphQLTestClientWithMongoDB()
+    testClientPublic = setupClient.testClientPublic
+    testClientPrivate = setupClient.testClientPrivate
+    dbAdapter = setupClient.dbAdapter
+
+    console.log('public', testClientPublic)
+  } catch (error) {
+    console.log('Error', error)
+    throw new Error('Error during test setup')
+  }
+})
+
+describe('Authors', () => {
+  describe('can be created/edited/deleted:', () => {
+    let authorIds: string[] = []
+    beforeEach(async () => {
+      const {mutate} = testClientPrivate
+      const authorInput: AuthorInput = {
+        name: 'JRR Tolkien',
+        slug: `tolkien-${authorIds.length}`,
+        bio: []
```

Comment:
I suggest you add some content to the bio. You can add an author with a bio through the editor and the copy the content from the database.